app/views/welcome/index.html.erb

```diff
@@ -1,12 +1,31 @@
 <div class="welcome">
   <link rel="stylesheet" href="./stylesheets/welcome">
   <div class="booke-image">
-    <div>
-      <%= render :partial => "/item_categories/category_select" %>
+    <div id="searchbar-container">
     </div>
     <div id="searchbar">
-      <div id="search-container">
-      </div>
+    <form action="/items" method="get"></form>
```

Comment:
Should the form elements be encased in the form tags?

Reply:
oops, fixing it now